docs/api/navigators/StackNavigator.md

```diff
@@ -118,7 +118,7 @@ All `navigationOptions` for the `StackNavigator`:
 
 - `title` - a title (string) of the scene
 - `header` - a config object for the header bar:
-  - `visible` - Boolean toggle of header visibility. Only works when `headerMode` is `screen`.
+  - `bar` - React Element to display as a header or `null` to hide it. Only works when `headerMode` is `screen`
```

Comment:
Yes, totally agree this is useful. It's pretty similar to `headerComponent` and I don't think we should have two configs doing very similar things.

> This is esp. useful to e.g. render an empty, transparent header on a screen that displays a full-screen image with just X icon.

That's just a single use case. What if you want to render previous screen title or something like that?